torch_frame/data/stats.py

```diff
@@ -49,15 +49,27 @@ def stats_for_stype(stype: torch_frame.stype) -> List['StatType']:
     def compute(self, ser: Series, sep: Optional[str] = None) -> Any:
         if self == StatType.MEAN:
             flattened = np.hstack(np.hstack(ser.values))
-            return np.mean(flattened).item()
+            finite_mask = np.isfinite(flattened)
+            if not finite_mask.any():
+                return np.nan
+            return np.mean(flattened[finite_mask]).item()
 
         elif self == StatType.STD:
             flattened = np.hstack(np.hstack(ser.values))
-            return np.std(flattened).item()
+            finite_mask = np.isfinite(flattened)
+            if not finite_mask.any():
+                return np.nan
```

Comment:
I am not sure whether it's better to just raise error here? nan stats seem not useful?

Reply:
That's true. I think we can keep it for now, but I added a note to potentially error out in the future.